docs/pages/hooks/useCreateNFT.mdx

```diff
@@ -104,14 +106,15 @@ Arguments:
 
 ```tsx
 {
-  standard: Standard,                           // can be Standard.ERC721 or Standard.ERC1155
+  chainId: Number,                              // id of the chain to mint on
+  collectionAddress: string,                    // address of the collection to mint on
   name: string,                                 // name of the NFT
   description: string,                          // description of the NFT
   content: File,                                // content file uploaded by the user
   preview?: File,                               // preview in the case of private or animation content uploaded by user
   isAnimation: boolean,                         // set to true if content file is a video. Require to set preview
   isPrivate: boolean,                           // set to true if content is private. Require to set preview.
-  amount?: number,                              // number of NFT to create. Require Standard.ERC1155
+  amount?: number,                              // number of NFT to create. Require ERC1155
```

Comment:
Could we rename `amount` to `supply` for clarity?
As there is breaking change on this function anyway, it's a good time to update params' name.

Reply:
Not really related to this PR but agree that's something that would improve and that can go in the same release

Reply:
https://github.com/liteflow-labs/liteflow-js/issues/105